x-pack/test/cases_api_integration/security_and_spaces/tests/common/internal/bulk_get_cases.ts

```diff
@@ -119,30 +103,15 @@ export default ({ getService }: FtrProviderContext): void => {
           ids: [caseOneUpdated.id, caseTwoUpdated.id],
         });
 
-        /**
-         * For performance reasons bulk_get does not
-         * return the comments
-         */
         expect(cases).to.eql([
-          { ...caseOneUpdated, comments: [], totalComment: 1, totalAlerts: 2 },
-          { ...caseTwoUpdated, comments: [], totalComment: 0, totalAlerts: 1 },
+          { ...getBulkGetCase(caseOneUpdated), totalComments: 1 },
+          { ...getBulkGetCase(caseTwoUpdated), totalComments: 0 },
```

Comment:
In `x-pack/plugins/cases/server/client/cases/bulk_get.ts` you remove totalAlerts(like discussed in my other comment) but what about this `comments: []`? Should we still look for it? I didn't see where it came from.

Reply:
Before it was possible to ask for the `comments` as a field (it will return an empty array though). Now that we explicitly set the returned fields, it is not needed anymore.